src/components/Navbar/index.tsx

```diff
@@ -1,5 +1,146 @@
+import { useEffect, useState } from 'react';
+import s from './style.module.scss';
+import CloseIcon from '@mui/icons-material/Close';
+import MenuIcon from '@mui/icons-material/Menu';
+import { AppBar, Container, Box, IconButton, Dialog, Toolbar, createTheme, ThemeProvider } from '@mui/material';
+import Link from 'next/link';
+
+const pages = [
+  {
+    name: 'About',
+    link: '/#about',
+  },
+  {
+    name: 'FAQ',
+    link: '/#faq',
+  },
+  {
+    name: 'Companies',
+    link: '/#companies',
+  },
+  {
+    name: 'Registration',
+    link: 'https://acmurl.com/space-registration',
+    externalLink: true,
+  },
+];
+
 const Navbar: React.FC = () => {
-  return <nav></nav>
-}
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#FFFFFF',
+      },
+      secondary: {
+        main: '#110f47',
+      },
+    },
+  });
+
+  const [mobileOpen, setMobileOpen] = useState(false);
+
+  const handleMobileOpen = () => {
+    setMobileOpen(true);
+  };
+
+  const handleMobileClose = () => {
+    setMobileOpen(false);
+  };
+
+  const [width, setWidth] = useState(0);
+  useEffect(() => {
+    setWidth(window.innerWidth);
+
+    window.addEventListener('resize', () => {
+      setWidth(window.innerWidth);
+    });
+  }, []);
+
+  if (width > 768) {
```

Comment:
It's not very pleasant how similar the two navbars are, but I guess the repetition is kind of necessary since the links need to be moved into a `Dialog` for mobile

![image](https://github.com/acmucsd/space-2022/assets/22133785/81d25a7f-3346-469a-aeaa-1056aab5861d)